bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Tree.java

```diff
@@ -5371,7 +5371,7 @@ public void setTopItem (TreeItem item) {
  * In a Tree without imageList, the indent also controls the chevron (glyph) size.
  */
 private void calculateAndApplyIndentSize() {
-	int indent = DPIUtil.autoScaleUpUsingNativeDPI(DEFAULT_INDENT);
+	int indent = DPIUtil.autoScaleUp(DEFAULT_INDENT);
```

Comment:
That won't work in multi monitor setup with different zoom levels. You always need to use the (scaleUp) method and passing the zoom to it, in this case the nativeZoom from the Tree